Tests/SwiftParserTest/AvailabilityTests.swift

```diff
@@ -121,8 +121,7 @@ final class AvailabilityTests: XCTestCase {
       substructure: Syntax(
         VersionTupleSyntax(
           major: .integerLiteral("10"),
-          minorPeriod: .periodToken(),
-          minor: .integerLiteral("0")
+          subVersions: VersionListSyntax([VersionTupleElementSyntax(period: .periodToken(), number: .integerLiteral("0"))])
```

Comment:
Do you even need to specify `periodToken` here? Shouldn’t it be defaulted because it’s non-optional? Same for the other test cases.